build_mesos

```diff
@@ -159,7 +159,13 @@ function build {(
   ./bootstrap
   mkdir -p "$build_dir"
   cd "$build_dir"
-  "$src_dir"/configure $(configure_opts)
+
+  # A bug in Autoconf 2.64 and earlier causes `AC_CONFIG_LINKS(src/a/b:src/c/d)`
+  # to create symlink 'src/a/b' that points to 'src/c/d' instead of pointing to
+  # '../../src/c/d'. This bug is triggered only when invoking configure script
+  # with an absolute path. Thus, we use relative invocation here.
+  # http://git.savannah.gnu.org/cgit/autoconf.git/commit/?id=13e3570
+  ../configure $(configure_opts)
```

Comment:
You are right. I totally missed the build_dir variable. I'll try to find a way to get a relative link and fix this one. Also, the `--src-dir` option has the same problem as using abspath to configure script :-(.